### PR TITLE
Use react on frontend

### LINKED ISSUE #6: Use react on frontend

### PR DESCRIPTION
**Pull request by tonitert**

### CODE REVIEW

File: .gitignore
Comment (jooakar) [inline .gitignore:1]: Do we need `.yarnrc.yml` or files in `.yarn/` or should they be ignored (or added)?

File: src/Localization.tsx
Comment (jooakar) [inline src/Localization.tsx:1]: Remove `lang.js` if it's replaced by this?

**Review by jooakar — APPROVED** [review]

File: src/Localization.tsx
Comment (tonitert) [reply src/Localization.tsx:1]: Sure, we don't need that anymore

File: .gitignore
Comment (tonitert) [reply .gitignore:1]: Added yarn files suggested by Yarn to be ignored, and .yarnrc.yml to .gitignore. If we need to include .yarnrc.yml later we can remove it from there

https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

**Pull request merged**